Topic 1/quickstart/src/test/java/MavenProyectTest/quickstart/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package MavenProyectTest.quickstart;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+    extends TestCase
+{
+    /**
+     * Create the test case
+     *
+     * @param testName name of the test case
+     */
+    public AppTest( String testName )
+    {
+        super( testName );
+    }
+
+    /**
+     * @return the suite of tests being tested
+     */
+    public static Test suite()
+    {
+        return new TestSuite( AppTest.class );
+    }
+
+    /**
+     * Rigourous Test :-)

```

Comment:
This class seems to be a copy ( including the comments :-) ) from https://github.com/LucasAguirre/java-bootcamp/pull/3/files#diff-80acd24b55c82729694f126286d65ec7R32